### PR TITLE
[Release 1.20] Hardened k8s 120

### PR DESCRIPTION
**Pull request by luthermonson**
#### Types of Changes ####

https://github.com/rancher/rke2/issues/1322
https://github.com/rancher/rke2/issues/1312

### CODE REVIEW

**Review by briandowns — APPROVED** [review]

**Review by galal-hussein — APPROVED** [review]

File: .drone.yml [hunk 1]
```diff
@@ -21,6 +21,20 @@ steps:
   - name: docker
     path: /var/run/docker.sock
 
+- name: package-images
+  image: rancher/dapper:v0.5.5
+  commands:
+    - docker pull --quiet rancher/hardened-build-base:v1.16.4b7
```
Comment (brandond) [inline .drone.yml:27]: The release-1.20 branch should be on hardened-build-base:v1.15.x (same version as upstream is using to build Kubernetes)

File: scripts/publish-image-runtime [hunk 2]
```diff
@@ -5,4 +5,5 @@ cd $(dirname $0)/..
 
 source ./scripts/version.sh
 
-docker image push ${REPO}/${PROG}-runtime:${DOCKERIZED_VERSION}-${GOARCH}
+docker image push ${REPO}/${PROG}-runtime:${DOCKERIZED_VERSION}-${GOOS}-${GOARCH}
+docker image push ${REPO}/${PROG}-runtime:${DOCKERIZED_VERSION}-windows-amd64
```
Comment (brandond) [inline scripts/publish-image-runtime:9]: We're not supporting Windows for 1.20, are we?

**Review by brandond — CHANGES_REQUESTED** [review]

File: scripts/publish-image-runtime (hunk 2 above)
Comment (luthermonson) [reply scripts/publish-image-runtime:9]: we are not, im still getting to this one

File: .drone.yml (hunk 1 above)
Comment (luthermonson) [reply .drone.yml:27]: that's news to me... thought it always needed to be latest.. no problem tho

File: .drone.yml [hunk 3]
```diff
@@ -21,6 +21,20 @@ steps:
   - name: docker
     path: /var/run/docker.sock
 
+- name: package-images
+  image: rancher/dapper:v0.5.5
+  commands:
+    - docker pull --quiet rancher/hardened-build-base:v1.15.8b5
```
Comment (brandond) [inline .drone.yml:27]: Same 1.15.x as upstream:

```suggestion
    - docker pull --quiet rancher/hardened-build-base:v1.15.14b5
```

**Review by brandond — CHANGES_REQUESTED** [review]

File: .drone.yml (hunk 3 above)
Comment (luthermonson) [reply .drone.yml:27]: https://github.com/rancher/rke2/blob/release-1.20/.drone.yml#L18 youre not talking about this upstream?

Comment (brandond) [reply .drone.yml:27]: No, but that'll need to change too as part of the release. I'm talking about [upstream kubernetes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.20.md#:~:text=is%20now%20built%20using%20go%201.15.14)

**Review by brandond — APPROVED** [review]

File: scripts/package-windows-images [hunk 4]
```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```
Comment (brandond) [inline scripts/package-windows-images:1]: Do we need this file at all if we're not going to do Windows on this branch?

Comment (luthermonson) [reply scripts/package-windows-images:1]: ya i thought i deleted that one, i got the others... good spot

**Pull request merged**